src/pages/AnimeSama/main.ts

```diff
@@ -0,0 +1,50 @@
+import { pageInterface } from '../pageInterface';
+
+export const AnimeSama: pageInterface = {
+  name: 'AnimeSama',
+  domain: 'https://anime-sama.fr',
+  languages: ['French'],
+  type: 'anime',
+  isSyncPage(url) {
+    return Boolean($('#playerDF').length);
+  },
+  sync: {
+    getTitle(_) {
+      return j.$('#titreOeuvre').text().trim();
+    },
+    getIdentifier(url) {
+      return utils.urlPart(url, 4) || '';
```

Comment:
Season 1 and 2 have the same identifier

https://anime-sama.fr/catalogue/3d-kanojo/saison2/vostfr/
https://anime-sama.fr/catalogue/3d-kanojo/saison1/vostfr/